doc/man3/SSL_write.pod

```diff
@@ -17,6 +18,15 @@ SSL_write_ex() and SSL_write() write B<num> bytes from the buffer B<buf> into
 the specified B<ssl> connection. On success SSL_write_ex() will store the number
 of bytes written in B<*written>.
 
+SSL_sendfile() writes B<size> bytes from offset B<offset> in the file file
+descriptor B<fd> to the specified SSL connection B<s>. This function provides
+efficient zero-copy semantics. SSL_sendfile() is available only when ktls is
+enabled, which can be checked by calling BIO_get_ktls_send() .
+t is provided here to allow users to maintain the same interface.
+For FreeBSD B<flags> is the same as the flags parameter to sendfile.
```

Comment:
I'm just wondering about the relevance of this since KTLS is linux only (at the moment). Presumably you are thinking that it eventually might be ported to other platforms - but this reference might imply it is already available on FreeBSD. Perhaps we should keep this simple and say something like "The meaning of flags is platforms dependent. Currently on Linux it is ignored" - or similar wording, without actually mentioning FreeBSD explicitly.

Reply:
I know that there are similar, but somewhat different patches in FreeBSD. I assume these would be upstreamed eventually.

I will update according to your suggestion.